src/components/editMeasure/testCases/components/editTestCase/qdm/populations/TestCasePopulation.tsx

```diff
@@ -100,8 +117,8 @@ const TestCasePopulation = ({
           {isTestCaseExecuted && populationResult ? (
             <ExpectActualInput
               id={`${populationResult?.id}-actual-cb`}
-              aria-labelledby={`${populationResult?.name}-actual`}
-              name={populationResult?.name}
+              aria-labelledby={`${population.name}${count}-actual`}
```

Comment:
Modify to human readable aria-label, concatenating the two governing labels,using the left row header and the top column header